examples/devserver/src/BUILD.bazel

```diff
@@ -0,0 +1,51 @@
+# For the notes in this file see:
+# https://github.com/bazelbuild/rules_nodejs/issues/84
+# https://github.com/bazelbuild/bazel/issues/1262
+# https://docs.google.com/document/d/1qPOUeoqDA3eWFFXS1shWX1FT3e4BQB8yuSMrfQL4QrA
+
+load("@build_bazel_rules_nodejs//internal/js_library/v2:js_library.bzl", "js_library")
+load("@build_bazel_rules_nodejs//:defs.bzl", "rollup_bundle")
+load("@build_bazel_rules_typescript//:defs.bzl", "ts_devserver")
+
+js_library(
+    name = "module",
+    srcs = ["module.js"],
+)
+
+js_library(
+    name = "test",
+    srcs = ["test.js"],
+    # Note: This rc file will only be found by babel when run in the devserver_example workspace
+    # but not in the top-level one, due to how bazel currently handels pathing
+    data = [
+        "//:.browserslistrc",
```

Comment:
this looks like a design flaw to me

if my js_library is depended on by two different root targets that provide binaries for two different platforms, how will those targets select the right outputs? The user would have to have parallel js_library trees right?

Reply:
Not sure I understand that comment. Yeah so the added `.browserslistrc` can change how babel transpiles the output js file - but you mean that instead of the `js_library` deciding on that the root target should do that? While I do like the idea I am not sure how that would possible.